lab2.js

```diff
@@ -127,24 +190,80 @@ function lastLetterSort(stringArray) {
     // this byLastLetter function is a "compare function"
     // And check out the "comparing strings" section  here:
     // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String
+    var aLength = a.length;
+    var bLength = b.length;
+
+    if (a.charAt((aLength - 1)) < b.charAt((bLength - 1))) {
+      return -1;
+    }
+    if (a.charAt((aLength - 1)) > b.charAt((bLength - 1))) {
+      return 1;
+    }
+    return 0;
```

Comment:
you can also return just a.CharAt... - b.charAt...
that will give you a positive #, negative #, or 0 - which is all that sort needs.